### PR TITLE
Documentation: improve cubes section

### PR DESCRIPTION
**Pull request by mbertrone**
This PR purpose is to improve `cubes` section, under `Documentation`

### CODE REVIEW

File: Documentation/cubes.rst [hunk 1]
```diff
@@ -32,8 +33,10 @@ A standard cube:
 A trasparent cube is a cube without any forwarding capability, such as a network monitor, NAT and a firewall.
 
 A trasparent cube:
```
Comment (acloudiator) [inline Documentation/cubes.rst]: `transparent` misspelled in the document.

Comment (mbertrone) [reply Documentation/cubes.rst]: Thanks Aasif! Now it should be fixed.

**Comment by frisso** [comment]
@acloudiator Unfortunately I'm unable to understand which changes have to be made. Can you please be more specific, so that we can fix it and merge the changes?

**Comment by mbertrone** [comment]
> @acloudiator Unfortunately I'm unable to understand which changes have to be made. Can you please be more specific, so that we can fix it and merge the changes?

All changes required by @acloudiator have been already solved.

**Pull request merged**